gamestonk_terminal/cryptocurrency/defi/terramoney_fcd_model.py

```diff
@@ -0,0 +1,273 @@
+"""Terra Money FCD model"""
+__docformat__ = "numpy"
+
+import textwrap
+from typing import Any
+from datetime import datetime
+import requests
+import pandas as pd
+from gamestonk_terminal.cryptocurrency.dataframe_helpers import (
+    denominate_number,
+    prettify_column_names,
+    replace_unicode,
+)
+
+GOV_COLUMNS = [
+    "submitTime",
+    "id",
+    "depositEndTime",
+    "status",
+    "type",
+    "title",
+    "Yes",
+    "No",
+]
+GOV_STATUSES = ["voting", "deposit", "passed", "rejected"]
+VALIDATORS_COLUMNS = [
+    "validatorName",
+    "tokensAmount",
+    "votingPower",
+    "commissionRate",
+    "status",
+    "uptime",
+]
+
+
+def _make_request(endpoint: str) -> dict:
```

Comment:
nice little helper 😄 , need to add that to terraengineer as well

Reply:
I think it could be even more generalized, and used for almost all API requests :)

Reply:
i 100% agree with this!